src/ecrc-frontend/src/components/composite/sideCards/SideCards.js

```diff
@@ -89,25 +89,14 @@ export default function SideCards({ type, sideCardLinks }) {
         If you have a BC Services Card, you must use it to log in to the online
         security verification page.
       </div>,
-      <div key="bcscLearnMore">
-        <a
-          href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/login-with-card"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <div className="side-card-link">
-            Learn more about the BC Services Card.
-          </div>
-        </a>
-      </div>,
       <div key="bcscEligibility" className="side-card-text">
         <br />
         Eligibility for a BC Services Card is the same as for the Medical
         Services Plan (MSP).
       </div>,
       <div key="bcscEligibilityLink">
         <a
-          href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/get-a-card"
+          href="https://www2.gov.bc.ca/gov/content?id=87EEAD6D19974459950AA7FF7F60AD54"
```

Comment:
I'm a bit surprised they don't have some kind of proxy for this link. If the id changed everyone had to update. If it was proxied then they just update the proxy and nobody has to update.